ui/src/components/CsvSourceMapping.vue

```diff
@@ -186,6 +191,11 @@ export default class CsvSourceMapping extends Vue {
     })
   }
 
+  async downloadSource (source: CsvSource): Promise<void> {
+    const downloadLink = await this.$store.dispatch('project/downloadCSV', source)
+    window.open(downloadLink)
```

Comment:
I don't think this belongs to the component. Please move `window.open` to `store/modules/project.ts`